samples/guide/src/main/java/okhttp3/recipes/CustomTrust.java

```diff
@@ -74,185 +153,19 @@ public void run() throws Exception {
         .build();
 
     try (Response response = client.newCall(request).execute()) {
-      if (!response.isSuccessful()) throw new IOException("Unexpected code " + response);
+      if (!response.isSuccessful()) {
+        Headers responseHeaders = response.headers();
```

Comment:
is this behavior change desired? printing headers only for failed responses?

Reply:
Yeah, mainly the interesting thing was the logo rather than headers.  Was there a point to the headers?

Reply:
I thought for the kotlin example it was more intresting to show the handshake peerCertificates.

```
          for (peerCertificate in response.handshake?.peerCertificates.orEmpty()) {
            println((peerCertificate as X509Certificate).subjectDN)
          }
```

Reply:
Good call.